3rdparty/libcxx/CMakeLists.txt

```diff
@@ -54,7 +54,7 @@ add_library(oelibcxx_cxx STATIC
     libcxx/src/vector.cpp
     )
 
-target_compile_options(oelibcxx_cxx PRIVATE -std=c++14)
+target_compile_options(oelibcxx_cxx PRIVATE -std=c++14 -Wno-error=conversion)
```

Comment:
Can you add a documentation line describing why we need to ignore the conversion warnings?